proposals/jib-cli-surface.md

```diff
@@ -0,0 +1,90 @@
+# Jib CLI Surface Specification
+
+Specification for a command structures exposed to users of the jib-cli
+
+## Usage
+`jib COMMAND [OPTIONS]`
+
+## Commands
+
+The jib cli exposes the following top level commands
+```
+build        build a container
+```
+
+## Options
+
+### Required
+```
+-t  --target <image-ref>  a target image reference, using the url schema defined in jib
+                          gcr.io/project/image (default is build to registry)
+                          registry://gcr.io/project/image
+                          docker://some-image-ref
+                          tar://relative/path.tar
+                          tar:///aboslute/path.tar
+```
+
+### Optional
+
+
+#### Build Config
+```
+    --application-cache <directory>        location of the application cache
+    --base-image-cache <directory>         location of the base image cache
+-b, --build-file <file>                    location of the build file (default <context>/jib.yaml)
+-c, --context <dir>                        location of the build context (default .)
+    --docker-config <directory>            location of docker configuration
+    --name <image-ref>                     image name to bake into tar file (only available when using -t tar://...)
```

Comment:
is this required or optional?

Reply:
Name is required for building "tar" images (https://github.com/GoogleContainerTools/jib/blob/master/jib-core/src/main/java/com/google/cloud/tools/jib/api/Containerizer.java#L104)

Reply:
updated text